src/main/java/com/dev/cinema/controller/Injector.java

```diff
@@ -0,0 +1,35 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.model.Role;
+import com.dev.cinema.model.User;
+import com.dev.cinema.service.RoleService;
+import com.dev.cinema.service.UserService;
+import javax.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class Injector {
```

Comment:
let's rename this class to DataInitializer and user annotation Component instead of the Controller

Reply:
done